packages/@okta/i18n/package.json

```diff
@@ -1,28 +1,28 @@
 {
-  "_from": "@okta/i18n@55.0.0-beta.gbcf6a23",
-  "_id": "@okta/i18n@55.0.0-beta.gbcf6a23",
+  "_from": "@okta/i18n@56.0.0-alpha.g3cf5471",
+  "_id": "@okta/i18n@56.0.0-alpha.g3cf5471",
```

Comment:
I see we are upgrading to an alpha version for now to test, but for some reason you are not updating the .properties files... any specific reason? I see only .json files been updated

Reply:
oops fixed it now.